operators/advanced_network/adv_network_types.h

```diff
@@ -173,7 +173,7 @@ enum class AnoMgrType {
 };
 
 static constexpr const char* ANO_MGR_STR__DPDK = "dpdk";
-static constexpr const char* ANO_MGR_STR__DOCA = "doca";
+static constexpr const char* ANO_MGR_STR__DOCA = "gpunetio";
```

Comment:
Should we change that string variable name?

Reply:
Yeah thought the same thing when reviewing again. Not necessary but trivial enough, did it here: https://github.com/nvidia-holoscan/holohub/compare/a3c2dee23181b2a342e5b014b409583cf98eed04..5a352e3a58763a5e9d92a6bbf75f716b2fba879f